src/Microsoft.Health.Fhir.Shared.Api/Controllers/ExportController.cs

```diff
@@ -97,6 +98,7 @@ public class ExportController : Controller
             [FromQuery(Name = KnownQueryParameterNames.TypeFilter)] string typeFilter,
             [FromQuery(Name = KnownQueryParameterNames.Format)] string formatName,
             [FromQuery(Name = KnownQueryParameterNames.IsParallel)] bool isParallel = true,
+            [FromQuery(Name = KnownQueryParameterNames.MaxCount)] uint maxCount = 0,
```

Comment:
QQ - why does `maxCount` default to zero on system level export but not default on the others?

Reply:
It is because the bottom parameters (typeParameter and idParameter) can't have default values. I moved those parameters to the top and set default values. I did test that the lack of default values didn't make a difference, but it is good for consistency.